fas2ipa/cli.py

```diff
@@ -171,6 +171,18 @@ def migrate_group(group, ipa):
     )
     try:
         ipa.group_add(name, **group_args)
+        if config.get("agreements"):
```

Comment:
You don't need that test, if the dict is empty the iteration over `.items()` will just do nothing.